src/transformStateWithClones.js

```diff
@@ -8,6 +8,35 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const statesArray = [];
```

Comment:
don't use data type names in variable names
```suggestion
  const states = [];
```